client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -155,6 +151,24 @@ class EmailProvidersComparison extends Component {
 		this.isMounted = false;
 	}
 
+	getDefaultExpandedState() {
+		const { shouldPromoteGoogleWorkspace } = this.props;
+
+		if ( shouldPromoteGoogleWorkspace ) {
+			return {
+				forwarding: false,
+				google: true,
+				titan: false,
+			};
+		}
+
+		return {
+			forwarding: false,
+			google: false,
+			titan: true,
+		};
+	}
+
```

Comment:
I'm thinking that maybe we are not going to promote forwarding subscription ever. Shall I remove that property?

Reply:
I'll leave that in there for now. We should consider that in another PR when the context is pretty clear.